r/R/dataset.R

```diff
@@ -360,6 +362,8 @@ IpcFileFormat <- R6Class("IpcFileFormat", inherit = FileFormat)
 #' - `$UseThreads(threads)`: logical: should the scan use multithreading?
 #' The method's default input is `TRUE`, but you must call the method to enable
 #' multithreading because the scanner default is `FALSE`.
+#' - `$BatchSize(batch_size)`: integer: Maximum row count of scanned record
+#' batches. Default is 64*2**10 rows.
```

Comment:
Can you make this a human-readable number?

Reply:
I was going for `64K`, is that better? Alternatively I can just write out the whole number

Reply:
`64K` is fine.